cmake-modules/AzureBuildTargetForCI.cmake

```diff
@@ -18,7 +18,7 @@ endmacro()
 # Adds a target to be built and also run during a CI run
 # - param service: The Azure SDK service group
 # - param target:  The name of the cmake target to be built
-macro(create_per_service_target_build_for_sample  service target)
+macro(create_per_service_target_build_for_sample  service target run_sample)
```

Comment:
Does it make sense to add this and have to maintain it througout all files. I'd rather this was set in this run place. Its ON for everything or OFF for everything. Right now it does work anywhere, but once its does we'd disable tests in specific places against bugs. 

I don't see us ever wanting this OFF for anything other than an active bug.

Reply:
CMAKE doesn't seem to have the concept of a "default" parameter. So we need to indicate if we want a sample to be run or not manually.

Right now it appears that about 50% of the samples which use this macro need to be disabled, and that will continue until November.

I believe our choice is (a) disable building the samples until November or (b) disable *running* the samples until November. I think that (b) is preferable.

Reply:
Ok, figured out how to make the final argument optional, you can now mark an individual sample as being disabled for execution.